.gitignore

```
@@ -12,3 +12,5 @@ coverage
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.env
```

Vite has some conventions for env variables. Local env files are named `.env.local` while versioned env files are named `.env`. What do you think about following these conventions?

https://vitejs.dev/guide/env-and-mode.html

```suggestion
*.local

```